eth-contracts/test/delegateManager.test.js

```diff
@@ -26,7 +27,7 @@ const testEndpoint4 = 'https://localhost:5009'
 const testEndpoint5 = 'https://localhost:5010'
 
 
-const INITIAL_BAL = _lib.audToWeiBN(1000000)
+const INITIAL_BAL = _lib.audToWeiBN(10000000)
```

Comment:
for some reason the proxy upgrade call fails without this change...not sure why @hareeshnagaraj

Reply:
but i don't think its a cause for concern?

Reply:
are you sure the proxy upgrade is failing..?

Reply:
i'm running locally and see 2 tests that perform the proxy upgrade - one is 
`Min delegator stake per SP - with DelegateManagerV2` and the other is
 `lets a service provider return to valid bounds when a delegator changes delegation (only works with DelegateManagerV2)`


Only the second one fails with the original initial balance value and even that is due to an erc transfer? 

```
       Delegation tests
         lets a service provider return to valid bounds when a delegator changes delegation (only works with DelegateManagerV2):
     Returned error: VM Exception while processing transaction: revert SafeERC20: low-level call failed -- Reason given: SafeERC20: low-level call failed.
```

Reply:
yeah this is exactly the failure i'm talking about

Reply:
that is unrelated to proxy though

Reply:
are you sure? it only happens on the proxy call i think

Reply:
nvm, resolved. was unrelated